Extension/src/Utility/Sandbox/interfaces.ts

```diff
@@ -0,0 +1,43 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { is } from '../System/guards';
+
+export interface ScriptError {
+    line: number;
+    column: number;
+    message: string;
+    file: string;
+    category: number;
+    code: number;
+    offset: number;
+}
+
+export interface CreateOptions {
+    /** the "filename" of the source to compile */

```

Comment:
Is there a reason `"filename"` is in quotes?

Reply:
It was probably because the filename isn't necessarily an actual file name, but may be any identifier.